packages/react-metaforms-antd/src/components/Multiselect.tsx

```diff
@@ -133,7 +128,7 @@ const Multiselect = React.forwardRef(({ options = [], value = [], onChange, plac
     <>
       {options.length > 5 && (
```

Comment:
Týhle podmínky bysme se asi mohli zbavit, když se ten search ovládá přes props

Reply:
tak je to tam